integration/js/utils/SdkBaseTest.js

```diff
@@ -241,6 +230,24 @@ class SdkBaseTest extends KiteBaseTest {
       await this.seleniumSessions[i].quit();
     }
   }
+
+  async printRunDetails(testResult) {
```

Comment:
nit: can we change the method name to something like "sessionRunDetails" to differentiate from the printRunDetails() in the WebdriverSauceLabs.js which is actually logging the session details ?